DataContent/DAL/Interfaces/ISmartphoneRepository.cs

```diff
@@ -0,0 +1,18 @@
+﻿using ItemLibrary;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DataContent.DAL.Interfaces
+{
+    public interface ISmartphoneRepository
```

Comment:
ISmartphoneRepository IComputerRepository interfaces are the same, maybe there you could use Generics? 
public interface IItemRepository<T> where T: Item

Reply:
Generic interfaces will be implemented in the next patch, for now we will leave it as is to not overengineer anything